dispatch/api/serializers.py

```diff
@@ -845,3 +849,46 @@ def update(self, instance, validated_data, is_new=False):
             instance.save_answers(answers, is_new)
 
         return instance
+
+class SubscriptionSerializer(DispatchModelSerializer):
+    """Serializes the Subscription model."""
+
+    endpoint = serializers.CharField(required=True, write_only=True)
+    auth = serializers.CharField(required=True, write_only=True)
+    p256dh = serializers.CharField(required=True, write_only=True)
+
+    class Meta:
+        model = Subscription
+        fields = (
+            'id',
+            'endpoint',
+            'auth',
+            'p256dh'
+        )
+
+    def create(self, validated_data):
+        # Create new Subscription instance
+        instance = Subscription()
+
+        instance.endpoint = validated_data.get('endpoint', instance.endpoint)
+        instance.auth = validated_data.get('auth', instance.auth)
+        instance.p256dh = validated_data.get('p256dh', instance.p256dh)
+
+        instance.save()
+
+        return instance
```

Comment:
This looks like the default functionality for a `create` method -- I'm not sure if you even need this.